packages/search/README.md

```diff
@@ -83,6 +82,20 @@ Note that you must pass the `keys` argument so the hook indexes those keys. Thos
 
 Beyond passing `keys` (and [its variations](https://fusejs.io/examples.html)), you can extend the hook by providing the `options` argument, which will extend the fuzzy search mechanism following [the options described by Fuse.js](https://fusejs.io/api/options.html).
 
-##### Setting an Initial Query
+##### Updating Query Setting an Initial Query
 
-It's also possible to set an initial query through the `initialQuery` argument. That'll be the the initial state for the returned `query` property and will be used as the initial search filter.
+You ["control"](https://reactjs.org/docs/uncontrolled-components.html) the query string
+passed to the hook. Updating the query string should be handled in the parent component
+before being passed in to the hook.
+
+```js
+const Component = () => {
+	const [ query, setQuery ] = useState( '' );
+	const searchResults = useFuzzySearch( {
+		data: [ 'site.wordpress.com', 'another.wordpress.com' ],
+		query: query,
+	} );
```

Comment:
The reason why query management was inside `useFuzzySearch` is that state management was implicit so you won't need to do it when building the interface. Either is fine to me.